Can't use plugin with java 7

Hi there,
tried to use the plugin with java 7 and it fails with the following error

```
[ERROR] Failed to execute goal at.makubi.maven.plugin:avrohugger-maven-plugin_2.11:1.1:generate-scala-sources (default) on project Boo: Execution default of goal at.makubi.maven.plugin:avrohugger-maven-plugin_2.11:1.1:generate-scala-sources failed: Unable to load the mojo 'generate-scala-sources' in the plugin 'at.makubi.maven.plugin:avrohugger-maven-plugin_2.11:1.1' due to an API incompatibility: org.codehaus.plexus.component.repository.exception.ComponentLookupException: at/makubi/maven/plugin/avrohugger/GeneratorMojo : Unsupported major.minor version 52.0
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>at.makubi.maven.plugin:avrohugger-maven-plugin_2.11:1.1
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/home/eplatonov/.m2/repository/at/makubi/maven/plugin/avrohugger-maven-plugin_2.11/1.1/avrohugger-maven-plugin_2.11-1.1.jar
[ERROR] urls[1] = file:/home/eplatonov/.m2/repository/org/codehaus/plexus/plexus-utils/3.0.22/plexus-utils-3.0.22.jar
[ERROR] urls[2] = file:/home/eplatonov/.m2/repository/org/apache/commons/commons-lang3/3.4/commons-lang3-3.4.jar
[ERROR] urls[3] = file:/home/eplatonov/.m2/repository/javax/enterprise/cdi-api/1.0/cdi-api-1.0.jar
[ERROR] urls[4] = file:/home/eplatonov/.m2/repository/javax/annotation/jsr250-api/1.0/jsr250-api-1.0.jar
[ERROR] urls[5] = file:/home/eplatonov/.m2/repository/org/eclipse/sisu/org.eclipse.sisu.inject/0.3.2/org.eclipse.sisu.inject-0.3.2.jar
[ERROR] urls[6] = file:/home/eplatonov/.m2/repository/org/codehaus/plexus/plexus-component-annotations/1.5.5/plexus-component-annotations-1.5.5.jar
[ERROR] urls[7] = file:/home/eplatonov/.m2/repository/io/spray/spray-json_2.11/1.3.2/spray-json_2.11-1.3.2.jar
[ERROR] urls[8] = file:/home/eplatonov/.m2/repository/org/scala-lang/scala-library/2.11.8/scala-library-2.11.8.jar
[ERROR] urls[9] = file:/home/eplatonov/.m2/repository/com/julianpeeters/avrohugger-core_2.11/0.14.0/avrohugger-core_2.11-0.14.0.jar
[ERROR] urls[10] = file:/home/eplatonov/.m2/repository/org/scala-lang/scala-compiler/2.11.8/scala-compiler-2.11.8.jar
[ERROR] urls[11] = file:/home/eplatonov/.m2/repository/org/scala-lang/modules/scala-xml_2.11/1.0.4/scala-xml_2.11-1.0.4.jar
[ERROR] urls[12] = file:/home/eplatonov/.m2/repository/org/scala-lang/modules/scala-parser-combinators_2.11/1.0.4/scala-parser-combinators_2.11-1.0.4.jar
[ERROR] urls[13] = file:/home/eplatonov/.m2/repository/org/apache/avro/avro/1.7.7/avro-1.7.7.jar
[ERROR] urls[14] = file:/home/eplatonov/.m2/repository/org/codehaus/jackson/jackson-mapper-asl/1.9.13/jackson-mapper-asl-1.9.13.jar
[ERROR] urls[15] = file:/home/eplatonov/.m2/repository/com/thoughtworks/paranamer/paranamer/2.3/paranamer-2.3.jar
[ERROR] urls[16] = file:/home/eplatonov/.m2/repository/org/xerial/snappy/snappy-java/1.0.5/snappy-java-1.0.5.jar
[ERROR] urls[17] = file:/home/eplatonov/.m2/repository/org/apache/commons/commons-compress/1.4.1/commons-compress-1.4.1.jar
[ERROR] urls[18] = file:/home/eplatonov/.m2/repository/org/tukaani/xz/1.0/xz-1.0.jar
[ERROR] urls[19] = file:/home/eplatonov/.m2/repository/org/apache/avro/avro-compiler/1.7.7/avro-compiler-1.7.7.jar
[ERROR] urls[20] = file:/home/eplatonov/.m2/repository/commons-lang/commons-lang/2.6/commons-lang-2.6.jar
[ERROR] urls[21] = file:/home/eplatonov/.m2/repository/org/apache/velocity/velocity/1.7/velocity-1.7.jar
[ERROR] urls[22] = file:/home/eplatonov/.m2/repository/commons-collections/commons-collections/3.2.1/commons-collections-3.2.1.jar
[ERROR] urls[23] = file:/home/eplatonov/.m2/repository/org/scala-lang/scala-reflect/2.11.8/scala-reflect-2.11.8.jar
[ERROR] urls[24] = file:/home/eplatonov/.m2/repository/org/codehaus/jackson/jackson-core-asl/1.9.13/jackson-core-asl-1.9.13.jar
[ERROR] urls[25] = file:/home/eplatonov/.m2/repository/com/eed3si9n/treehugger_2.11/0.4.1/treehugger_2.11-0.4.1.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
[ERROR] 
[ERROR] -----------------------------------------------------
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginContainerException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :Boo
```

looks like this line https://github.com/makubi/avrohugger-maven-plugin/blob/master/pom.xml#L17 is the cause.
Is it possible to make it work for java 7 and 8? 

Thanks

--- a/pom.xml
+++ b/pom.xml
@@ -14,7 +14,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <java.version>1.8</java.version>
+        <java.version>1.6</java.version>
         <scala.binary.version>2.11</scala.binary.version>
         <scala.version>${scala.binary.version}.8</scala.version>
 

